modules/update-other-images.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-44.adoc
+:_module-type: PROCEDURE
+[id="update-other-images_{context}"]
+= Updating other images
+
+[role="_abstract"]
+If you are upgrading {product-title} version number between 3.0.44 and 3.0.54, you must update the Sensor and Collector images.
+
+[WARNING]
+====
+Updating images is only required when you are upgrading from {product-title} version number between 3.0.44 and 3.0.54. Otherwise, skip the instructions in this section.
```

Comment:
There are still a handful of this wording with "number" that could be changed to remove "number" and add "a"